### ISSUE TITLE
0.1.2.0 version can not debug/run war

### ISSUE THREAD

**Issue by cooml** [adashen/vscode-tomcat#329]
0.1.2.0 version can not debug/run war

**Comment by KoMaBeLu** [comment]
Having the same issue

**Comment by JoaquinBenitez** [comment]
I think the command to Debug a war has stopped working.

You can still run or debug if you do: **Right click** -> **Debug on tomcat server**, on the **folder that has the same name as your war**

E.g: Inside target I have some folders, one of them API, and API.war, and API.war doesn't work when trying to debug it, so right click on API folder and Debug on tomcat server. That worked for me.

**Comment by Walker088** [comment]
Having the same issue here, I guess the only work around now would be downgrade to the previous version.

Try uninstall the plugin, and by the following idea:
```bash
$ wget https://github.com/adashen/vscode-tomcat/releases/download/v0.11.2/vscode-tomcat-0.11.2.vsix
$ code --install-extension vscode-tomcat-0.11.2.vsix
```

after that I'm now able to use it lol

btw, right click neither on the war file or the folder work for me

**Comment by sledwa** [comment]
same issue.  thanks @Walker088 , your solution worked for me.

**Comment by KoMaBeLu** [comment]
@Walker088 this is only useful if you don't rely on the feature introduced by PR #318 for setting a custom JAVA_HOME, I had to downgrade to my own custom implementation of this extension to be able to work

**Comment by cooml** [comment]
Thanks @JoaquinBenitez ,  your solution worked for me.
@sledwa, I think we don't need downgrade to the previous version. use folder instead of war file debug on  tomcat server

**Comment by Eskibear** [comment]
Please try if below fix works, extract and install the .vsix file.

[vscode-tomcat-0.12.0-fix-debug-war.zip](https://github.com/adashen/vscode-tomcat/files/7438466/vscode-tomcat-0.12.0-fix-debug-war.zip)

**Comment by Walker088** [comment]
> Please try if below fix works, extract and install the .vsix file.
> 
> [vscode-tomcat-0.12.0-fix-debug-war.zip](https://github.com/adashen/vscode-tomcat/files/7438466/vscode-tomcat-0.12.0-fix-debug-war.zip)

The patch fix both on my Ubuntu and Windows WSL machines

**Comment by Eskibear** [comment]
@Walker088 Thank you for the feedback. I've already shipped it with v0.12.1. Hope it will unblock you guys.

**Comment by chunyisong** [comment]
v0.12.1 version still can not debug/run war file on win10 vscode!
Only on work on the folder.
